src/test/java/net/dv8tion/jda/test/components/ContainerTest.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2015 Austin Keener, Michael Ritter, Florian Spieß, and the JDA contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.dv8tion.jda.test.components;
+
+import net.dv8tion.jda.internal.components.container.ContainerImpl;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+
+public class ContainerTest
+{
+
+    @Test
+    void testEmptyContainerThrows()
+    {
+        Assertions.assertThatIllegalArgumentException()
```

Comment:
Minor: If you want you can extend `ChecksHelper` with a `assertCollectionChecks` to make this possible:

```java
assertCollectionChecks("Components", (list) -> new ContainerImpl(-1, list, false, null))
  .checksNotNull()
  .checksNotEmpty();
```